tests/test_match_import.py

```diff
@@ -133,8 +158,1836 @@ def test_load_match(self):
+            "info(scoreFilePath,/path/to/dataset/Chopin_op10_no3.musicxml)."
+        )
+        midiFileName_line = "info(midiFileName,Chopin_op10_no3_p01.mid)."
+        midiFilePath_line = (
+            "info(midiFilePath,/path/to/dataset/Chopin_op10_no3_p01.mid)."
+        )
+        audioFileName_line = "info(audioFileName,Chopin_op10_no3_p01.wav)."
+        audioFilePath_line = (
+            "info(audioFilePath,/path/to/dataset/Chopin_op10_no3_p01.wav)."
+        )
+        audioFirstNote_line = "info(audioFirstNote,1.2345)."
+        audioLastNote_line = "info(audioLastNote,9.8372)."
+        composer_line = "info(composer,Frèdéryk Chopin)."
+        performer_line = "info(performer,A. Human Pianist)."
+        midiClockUnits_line = "info(midiClockUnits,4000)."
+        midiClockRate_line = "info(midiClockRate,500000)."
+        approximateTempo_line = "info(approximateTempo,98.2902)."
+        subtitle_line = "info(subtitle,Subtitle)."
+
+        matchlines = [
+            version_line,
+            piece_line,
+            scoreFileName_line,
+            scoreFilePath_line,
+            midiFileName_line,
+            midiFilePath_line,
+            audioFileName_line,
+            audioFilePath_line,
+            audioFirstNote_line,
+            audioLastNote_line,
+            composer_line,
+            performer_line,
+            midiClockUnits_line,
+            midiClockRate_line,
+            approximateTempo_line,
+            subtitle_line,
+        ]
+
+        for i, ml in enumerate(matchlines):
+            mo = MatchInfoV1.from_matchline(ml)
+            # assert that the information from the matchline
+            # is parsed correctly and results in an identical line
+            # to the input match line
+            basic_line_test(mo, verbose=i == 0)
+            self.assertTrue(mo.matchline == ml)
+
+        # An error is raised if parsing the wrong version
+        try:
+            mo = MatchInfoV1.from_matchline(ml, version=Version(0, 1, 0))
+            self.assertTrue(False)  # pragma: no cover
+        except ValueError:
+            self.assertTrue(True)
+
+        # The following lines should result in an error
+        try:
+            # This line is not defined as an info line and should raise an error
+            notSpecified_line = "info(notSpecified,value)."
+
+            mo = MatchInfoV1.from_matchline(notSpecified_line)
+            self.assertTrue(False)  # pragma: no cover
+        except ValueError:
+            # assert that the error was raised
+            self.assertTrue(True)
+
+        try:
+            # wrong value (string instead of integer)
+            midiClockUnits_line = "info(midiClockUnits,wrong_value)."
+
+            mo = MatchInfoV1.from_matchline(midiClockUnits_line)
+            self.assertTrue(False)  # pragma: no cover
+        except ValueError:
+            # assert that the error was raised
+            self.assertTrue(True)
+
+        try:
```

Comment:
Is trying except clauses a valid practice for tests? What if it fails for a different `ValueError` and you accept the test.